packages/vm/Cargo.toml

```
@@ -29,6 +29,10 @@ cranelift = ["wasmer/cranelift"]
 # See https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options
 bench = false
 
+[[example]]
+name = "compile_module"
```

To me this sounds like a compiler, which should have an output. But the only thing we want here is checking contracts. So what about

```suggestion
name = "check_contract"
```